src/test/java/training/sign_up/sign_up.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Register a new user
+
+  Background:
+    * url someUrl
+
+  Scenario: Register successfully
+
+    * def username = register.username
+    * def email = register.email
+    * def password = register.password
```

Comment:
think about in the running of the tests. If you set a specific user to run this feature, you will need to modify the karate-config file to run the test every time. Try to generate a random mail in order to won't have this problem.